source/code-of-conduct/index.html.haml

```diff
@@ -1,44 +1,39 @@
 ---
-title: Brug - Code of Conduct - Ruby Belgium
+title: Code of Conduct - Ruby Belgium
 ---
 
-%header.header
+%header
```

Comment:
I don’t think using %section.alternate is not very predictable. It mean you now have to always increase the specificity by adding the html tag in order to differenciate 
```
li.alternate
section.alternate
header.alternate
…
```
But again, even if you would like to go that way, please make sure it’s consistent for others to use and don’t add different conventions at the same time.
ps: if we start not to use a naming scheme we have to be extra careful about class naming event more. Simply using ```.alternate``` would be a good thing to avoid.